src/components/TableResize.js

```diff
@@ -94,21 +94,32 @@ class TableResize extends React.Component {
   };
 
   onResizeStart = (id, e) => {
-    this.setState({ isResize: true, id, startPosition: e.clientX });
+    this.setState({ isResize: true, id });
   };
 
   onResizeMove = (id, e) => {
-    const { startPosition, isResize, resizeCoords } = this.state;
-
+    const { isResize, resizeCoords } = this.state;
+    const fixedMinWidth = 100;
+    const idNumber = parseInt(id);
+    const finalCells = Object.entries(this.cellsRef);
+    if (idNumber >= finalCells.length) return;
```

Comment:
Can you explain what situation would cause us to have an id column that is the same or greater than our total number of columns?

Reply:
made mistake here - should be length-1 - we should not process moving for last resizer.